pkg/controller/controller_serviceplan.go

```diff
@@ -39,9 +43,17 @@ func (c *controller) servicePlanAdd(obj interface{}) {
 // primarily reconciled in a separate flow when a ClusterServiceBroker is
 // reconciled.
 func (c *controller) reconcileClusterServicePlanKey(key string) error {
-	// currently, this is a no-op.  In the future, we'll maintain status
-	// information here.
-	return nil
+	plan, err := c.servicePlanLister.Get(key)
+	if errors.IsNotFound(err) {
```

Comment:
Check if `err != nil`?
I see 2 options:
1. Enclose both error checks into a single `if err != nil` code block:
```go
if err != nil {
  if errors.IsNotFound(err) {
    ...
    return nil
  }
  ...
  return err
}
```
2. Or you can reverse the order and do first:
```go
if err == nil {
  return c.reconcileClusterServicePlan(plan)
}
```
and then handle the errors.

Reply:
this actually matches exactly what we do in other files